src/main/resources/portal.properties.EXAMPLE

```diff
@@ -194,6 +194,9 @@ oncokb.token=
 # Enable merging of OncoKB icons by default
 # oncokb.merge_icons_by_default=true
 
+# Enable OncoKB menu settings "cBioPortal>=" and "COSMIC>=" (true, false)
+show.oncokb.cbiocosmic=true
```

Comment:
Correct names and separate cbioportal and cosmic.

Reply:
fixed